index.html

```diff
@@ -12,6 +13,485 @@
     <link rel="stylesheet" type="text/css" media="screen" href="css/main.css" />
   </head>
   <body>
-    
+    <header>
+      <div class="header">
+        <div>
+          <span><i class="fas fa-bars"></i></span>
+          <span><i class="fas fa-search"></i></span>
+          <span class="title">SPACE & COSMOS</span>
+        </div>
+        <div>
+          <img src="images/220px-The_New_York_Times_logo.png" alt="The New York Times"/>
+        </div>
+        <div>
+          <button>Subscribe Now</button>
+          <button>Log In</button>
+        </div>
+      </div>
+    </header>
+
+    <div class="float-section">
+        <div class="grid-images">
+            <h6><strong>Editors' Picks</strong></h6>
+            <hr>
+            <a class="image-text" href="#"><img src="images/glitter.jpg" alt="" align="left">The Meaning of the Scene: Sean Spicer Repents</a>
+            <a class="image-text" href="#"><img src="images/bugger.jpg" alt="" align="left">Bigger, Saltier, Heavier: Fast Food Since 1986 in 3 Simple Charts</a>
+            <a class="image-text" href="#"><img src="images/xx.jpg" alt="" align="left">‘XX’? But I Hardly Know Her! You watch the movie</a>
+        </div>
+    </div>
+    <div class="float-section-signup">
+        <h5><strong>Sign up for Science Times</strong></h5>
+        <p><span class="signup-text">We'll bring you stories that capture the wonders of the human body, nature and the cosmos.</span></p>
+        <button>SIGN UP</button>
+    </div>
+
+    <div class="main-content">
+      <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+      <div class="photo">
```

Comment:
Here is that big space

Reply:
Ok, thanks

Reply:
Fixed, now it's similar to the real site.